docs/pages/machine-id/guides/github-actions.mdx

```diff
@@ -32,116 +32,14 @@ Actions runners as well as GitHub Enterprise Server.
 - A GitHub repository with GitHub Actions enabled. This guide uses the example `gravitational/example`
 repo, however this value should be replaced with your own unique repo.
 
-## Step 1/3. Create a join Token for GitHub Actions
+## Step 1/3. Creating a join token for GitHub Actions
```

Comment:
This is a regression. For better SEO we want to avoid 'ing' verbs in headers.

Reply:
Sorry ! I modified this to -"ing" to match the rest of the guide as written by the rest of the docs team. I'll raise a new PR changing them all to non-"ing".